### ISSUE TITLE
[Mac OS] Download of a folder results in a corrupt zip file.

### ISSUE THREAD

**Issue by SergioBertolinSG** [owncloud/core#13669]
### Steps to reproduce

1. Create a folder with several files
2. Download the folder clicking on download.

### Expected behaviour
You can download the folder and open the zip file.

### Actual behaviour
Zip file downloaded is corrupt.

### Server configuration
**Operating system**:
Ubuntu 14.04 LTS

**Web server:**
Apache

**Database:**
MySQL

**PHP version:**
5.5.9

**ownCloud version:** (see ownCloud admin page)
{"installed":true,"maintenance":false,"version":"8.0.0.1","versionstring":"8.0 alpha 1","edition":""}

**Updated from an older ownCloud or fresh install:**
fresh

**List of activated apps:**
none

**The content of config/config.php:**

```
Insert your config.php content here
(Without the database password and passwordsalt)
```

**Are you using external storage, if yes which one:** local/smb/sftp/...
no

**Are you using encryption:** no

### Client configuration
**Browser:**
Chrome, Firefox

**Operating system**
Mac OS

Working fine in Windows and Linux.

**Comment by DeepDiver1975** [comment]
known issue - duplicate https://github.com/McNetic/PHPZipStreamer/issues/16

**Comment by SergioBertolinSG** [comment]
@DeepDiver1975  I find this issue really annoying, every mac os user can't download a folder without using desktop client. Will this be fixed soon?

**Comment by McNetic** [comment]
Just for the sake of completeness: The issue is known. The problem is not a currupt zip file, but rather OSX not supporting two widely spread zip features 10 respective 23 years old (namely zip64 support and streamed zip support). The workaround will probably be delivering tar files to OSX clients sometime in the future.

**Comment by michaelstingl** [comment]
@McNetic Probably not fixed in Mac OS X El Capitan :crying_cat_face: :crying_cat_face: :


![cursor_and_os_x_10_11](https://cloud.githubusercontent.com/assets/214010/9859116/ed5560f0-5b24-11e5-8826-66e9bb7bd193.png)
(Version 10.11 Beta 8)

**Comment by compuguy** [comment]
@McNetic @michaelstingl Not fixed in os x 10.11 (as of July 2016).

**Comment by RealRancor** [comment]
@compuguy Please create a new issue providing more information as the issue here and in #19226 was confirmed to be fixed in 8.1.4/8.2.0 with https://github.com/owncloud/core/pull/19318
